### ISSUE TITLE
Error tagging with pre-annotated data for document annotation

### ISSUE THREAD

**Issue by sthpravin** [DataTurks/DataTurks#17]
After uploading the pre-annotated data for document annotation and tagging the data, the annotations didnot saved incorrectly after marking them as done.

**Comment by IssamAssafi** [comment]
Do you still have this issue?

**Comment by cadet354** [comment]
yep, the problem with React code, to server sends only changed information but must be overall.
